src/renderer/sw_engine/tvgSwFill.cpp

```diff
@@ -841,6 +859,7 @@ void fillReset(SwFill* fill)
         fill->ctable = nullptr;
     }
     fill->translucent = false;
+    fill->useSolid = false;
```

Comment:
see `translucent`, it doesn't say `useTranslucent`.

please try to keep the consistency. Words that are not particularly important can be omitted.